tests/kernel/semaphore/semaphore/src/main.c

```diff
@@ -542,16 +542,16 @@ void test_sem_measure_timeouts(void)
 	/* With timeout of 1 sec */
 	start_ticks = k_uptime_get();
 
-	ret_value = k_sem_take(&simple_sem, SECONDS(1));
+	ret_value = k_sem_take(&simple_sem, K_SECONDS(1));
 
 	end_ticks = k_uptime_get();
 
 	zassert_true(ret_value == -EAGAIN,
 		     "k_sem_take failed when its shouldn't have");
 
-	zassert_true((end_ticks - start_ticks >= SECONDS(1)),
+	zassert_true((end_ticks - start_ticks >= K_SECONDS(1)),
 		     "time missmatch expected %d ,got %d\n",
-		     SECONDS(1), end_ticks - start_ticks);
+		     K_SECONDS(1), end_ticks - start_ticks);
```

Comment:
The rest of the PR looks good, but are you sure this part is correct? It looks like here the code is dealing with ticks and not milliseconds.

Reply:
It's ms.  The "ticks" are the result of calling k_uptime_get().  This seems to have been code from before the ticks->ms conversion in the API (IMHO a mistake, but too late now) that got updated but never renamed.

Reply:
Thanks for the clarification, not a show-stopper then, however would be good to rename these